telos-frontend/src/components/steam/WidgetSteam.module.css

```diff
@@ -0,0 +1,3 @@
+.cross {
```

Comment:
Where is this style class used? I can't really see where an import of this class and this id is used

Reply:
Right, I forgot to delete that file Nice spotting!